spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  let(:user) {FactoryBot.build :user, email: "BUIVANSAOBG@GMAIL.COM"}
+
+  describe "#email_downcase" do
```

Comment:
so sánh describe, context, it

Reply:
describe: tạo 1 cụm các trường hợp test, ví dụ khai báo cho các method
context: cũng tạo nhóm các trường hợp test, nhưng phân theo các trường hợp. ví duj: context "when user logged in"...
it: mô tả chi tiết trường hợp test cụ thể ah